docs/reference/objects.mdx

```diff
@@ -1667,6 +1701,20 @@ The status for a transaction that was squeezed out of the transaction pool.
 
 The reason why the transaction was squeezed out.
 
+## `StateTransitionBytecode`
+
+The bytecode of a state transition upgrade.
+
+**fields:**
+
+`root`: [`[HexString!]`](/docs/reference/scalars/#hexstring)
+
+The merkle root of the new state.
```

Comment:
@xgreenx correct me if I'm wrong, but this looks incorrect to me. This root should be the merkle root of the state transition bytecode, not the actual state.

Reply:
Yes, it is merle root of the STF